autoload/coqtail.vim

```diff
@@ -349,6 +349,24 @@ function! coqtail#start(...) abort
     call coqtail#panels#init()
     call coqtail#panels#open(0)
 
+    " Define commands and mappings for Goal buffer
+    let l:panel = coqtail#panels#switch(g:coqtail#panels#goal)
+    if l:panel != g:coqtail#panels#none
+      " Switched to goal panel
+      call s:commands(1)
+      call s:mappings()
+      call coqtail#panels#switch(l:panel)
+    endif
```

Comment:
This could be moved into `s:init` in autoload/coqtail/panels.vim to avoid the extra buffer switching. This also requires making `s:commands` and `s:mappings` global functions (e.g., `g:coqtail#commands`).

Reply:
I agree